lib/tasks/counter_metrics.rake

```diff
@@ -9,13 +9,36 @@ namespace :counter_metrics do
     ImportCounterMetrics.import_requests(args.csv_path)
   end
 
-  # bundle exec rake counter_metrics:import_requests['pittir.hykucommons.org','spec/fixtures/csv/pittir-downloads.csv']
   desc 'import historical counter investigations'
+  # bundle exec rake counter_metrics:import_requests['pittir.hykucommons.org','spec/fixtures/csv/pittir-downloads.csv']
```

Comment:
Would it make sense to move the example comments up to this area as well?

I also think comments should happen before the `desc` declaration. 

```ruby
##
# inline comment
desc "Terse description"
task ...
```